samples/snippets/src/main/java/com/examples/documentai/BatchParseForm.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.examples.documentai;
+
+// [START documentai_batch_parse_form]
```

Comment:
Nit: `Beta` to end of file name add `_beta` to region tag. (check with eric for this and other files)
- BatchParseFormBeta.java
- document_ai_batch_parse_form_beta